src/Entity/LreArea.php

```diff
@@ -48,4 +57,35 @@ public function setReferentTag(?ReferentTag $referentTag): void
     {
         $this->referentTag = $referentTag;
     }
+
+    public function isAllTags(): bool
+    {
+        return $this->allTags;
+    }
+
+    public function setAllTags(bool $allTags): void
+    {
+        $this->allTags = $allTags;
+    }
+
+    /**
+     * @Assert\Callback
+     */
+    public function validate(ExecutionContextInterface $context, $payload)
+    {
+        /** @var LreArea $lreArea */
+        $lreArea = $context->getValue();
+
+        if (!$lreArea->isAllTags()) {
```

Comment:
potentially we can have `LreArea` without `refTag` and without the option `AllTags` and it will be valid 🤔

Reply:
no, if this is the case, so the listener [here](https://github.com/EnMarche/en-marche.fr/pull/4663/files/9162aab36799d8945d1cd2bc8a77cae63a06b996#diff-e8cde86e76cde95a03e74a240541a646R33) will remove the lreArea to the adherent, like before

Reply:
I meen this: 
```php
$o = new LreArea();
$errors = $validator->validate($o);

// will output `int(0)`
var_dump(count($errors));
```